docs/auth-revoke-consents.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Revoke consent
+topic: auth
+type: how-to
+---
+
+# Delete user
+
+We support a flow where you can initiate the user to remove access to your application.
+
+This should be used when a user choose to remove the application within your app.  
+
+## Flow
+![](~/assets/images/auth-revoke-session.svg)
+
+1. Application requests a uninstall link.
+2. Application redirects the user to the given uri which opens a page hosted by WGTwo ID.  
+3. User confirms deleting the application by clicking a button in the page.
+4. User is redirected to a success page.
+
+In Step 2, the application may optionally include a `redirect_uri` and `state` query parameter.
```

Comment:
I think this need to change; how does the Application wipe the user from the app database if the app doesn't provide a redirect uri? Also we should indicate that the `state` param is used to make sure the user really does want to uninstall/wipe out the user data, that no malicious actor has opened `https://voicebox.cc/wipe-this-user` in an iframe.